guis/qml/qml/MainQML.qml

```diff
@@ -0,0 +1,60 @@
+import QtQuick.Controls 2.2
+import QtQuick 2.2
+import QtWebEngine 1.7
+import QtWebSockets 1.0
+import QtMultimedia 5.8
+
+    WebEngineView {
+    property QtObject request
+    property QtObject wsClient
+    property var handler
+    property var contentType
+    property var activeTransfer
+    property var selectionType
+    property var requestContentHub : false
+    //property string url
```

Comment:
Will these comments later be used, and if so when?

If they are only there because they are not-yet removed, I would vote to remove them before merging.